README.md

```diff
@@ -8,25 +8,25 @@
 
 ### Features
 
-- Integrates [detekt](https://github.com/arturbosch/detekt) for code analysis
-- Default quality profile `detekt active` (80 rules) and `detekt all` (164 rules)
-- Syntax highlighting
-- Supports SonarQube 6.7.7 and up
-- Supports detekt's `yaml config` and `baseline.xml` (also `path filters`)
-- Jacoco support
-- Lines of code and complexity metrics
+- Integrates [detekt](https://github.com/arturbosch/detekt) for static code analysis
+- Default quality profiles `detekt active` (80+ rules) and `detekt all` (164+ rules)
+- Supports SonarQube 7.9.3+
+- Supports detekt's `yaml config`, `baseline.xml` and `excludes`
+- Seamless integration with official SonarKotlin (no redundant features)
 
 ### Usage
 
-This sonar-kotlin is not the official sonarqube plugin.
-It was released before the official plugin and uses the same plugin key 'kotlin'.
-That means you can't use this plugin together with the official one.
+#### Sonar Update Center
 
-- `git clone https://github.com/arturbosch/sonar-kotlin`
-- `cd sonar-kotlin`
-- `mvn package`
-- `cp target/sonar-kotlin-[enter_version].jar $SONAR_HOME/extensions/plugins`
-- `cd $SONAR_HOME/bin/[your_os]`
+- TODO
```

Comment:
I guess the upload happens after v2.0 is released, right?

Reply:
Yep. It's quite a procedure ... registering in a forum and writing stuff xD